### PR TITLE
Update operators.md

### PR DESCRIPTION
**Pull request by keyem4251**
I found code comment 「AFTER」is uppercase but line 134 and line 142 「before」is lower case.
So  unified code comment uppercase and lowercase.

### CODE REVIEW

File: src/language/operators.md [hunk 1]
```diff
@@ -135,15 +135,15 @@ b = ++a; // Increment a before b gets its value.
...
+b = a++; // Increment a after b gets its value.
 assert(a != b); // 1 != 0
 
 a = 0;
 b = --a; // Decrement a before b gets its value.
 assert(a == b); // -1 == -1
 
 a = 0;
-b = a--; // Decrement a AFTER b gets its value.
+b = a--; // Decrement a after b gets its value.
```
Comment (parlough) [inline src/language/operators.md:146]: This code snippet is actually checked against an actual `.dart` file. You'll have to update the comments there as well: https://github.com/dart-lang/site-www/blob/main/examples/misc/test/language_tour/operators_test.dart#L59-L67

**Review by parlough — CHANGES_REQUESTED** [review]
Thanks for making this consistent! One little adjustment will be needed to get the build to pass.

Let me know if you have any issues doing so.

File: src/language/operators.md (hunk 1 above)
Comment (keyem4251) [reply src/language/operators.md:146]: Thank you reviewing.
I'll modify dart code.

Comment (keyem4251) [reply src/language/operators.md:146]: I updated the code comments  in operators_test.dart.
Please confirm change [088a99](https://github.com/dart-lang/site-www/pull/4759/commits/d088a99f02d92767d39d51bc1060997117be67cb)

Comment (parlough) [reply src/language/operators.md:146]: Yep, looks good, thanks!

**Review by parlough — APPROVED** [review]
Thanks for improving the consistency here and adjusting the base code as well. Looks good to me!

**Comment by keyem4251** [comment]
@parlough 
Thank you approve pull request.
But build/test(beta) is failed.I don’t understand this reason.
Do you know why test is failed?

**Comment by parlough** [comment]
The failure is due to a separate issue. I prepared a fix in https://github.com/dart-lang/site-www/pull/4763. Once I land that, I'll land your changes :)

Thanks!

**Comment by keyem4251** [comment]
Thanks. Understood!

**Pull request merged**